ouroboros-consensus/src/Ouroboros/Consensus/HardFork/History/Qry.hs

```diff
@@ -533,7 +545,7 @@ instance Show (ClosedExpr a) where
     where
       go :: Int  -- How many variables are already in scope?
          -> Int  -- Precedence
-         -> Expr Var a -> ShowS
+         -> Expr Var b -> ShowS
```

Comment:
Needed because I enabled `ScopedTypeVariables` for `runQuery`. The `a` in this `go` is *not* the `a` from `Show (ClosedExpr a)`.